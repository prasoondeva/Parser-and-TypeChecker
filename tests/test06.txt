{
	x1 , x2 : INT;
	y : INT;
	x1 = y;
	y = + x1 x2;
}