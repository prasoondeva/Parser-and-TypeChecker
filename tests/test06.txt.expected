UNINITIALIZED y 4
UNINITIALIZED x2 5